yt_idefix/data_structures.py

```diff
@@ -724,3 +797,241 @@ def _validate_units_override_keys(cls, units_override):
             )
 
         super(cls, cls)._validate_units_override_keys(units_override)
+
+
+class PlutoXdmfDataset(PlutoVtkDataset):
+    _index_class = PlutoXdmfHierarchy
+    _field_info_class = PlutoXdmfFields
+    _dataset_type = "pluto-xdmf"
+    _version_regexp = re.compile(r"\d+\.\d+\.?\d*[-\w+]*")
+    _required_header_keyword = "PLUTOXdmf"
+    _default_definitions_header = "definitions.h"
+    _default_inifile = "pluto.ini"
+    _field_offset_index = None
+
+    def _parse_parameter_file(self):
+        grid_file = os.path.join(os.path.dirname(self.parameter_filename), "grid.out")
+        self.parameter_filename[:-2] + "xmf"
```

Comment:
I meant to keep a variable for the xmf file location in case it is used later. But I think since I'm currently not using it, I missed the assignment. Let this be assigned to a variable as xmf files have useful info that can be used later for something.